_posts/2021-09-01-cockpit-252.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 252
+author: skobyda
+date: '2021-09-01'
+tags: cockpit
+slug: cockpit-252
+category: release
+summary: 'Remotectl utility dropped, introduced package version listing, improved presentation of encryption of a filesystem'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 252:
+
+
+## Webserver: Drop remotectl utility
+
+[Cockpit 241](https://cockpit-project.org/blog/cockpit-241.html) already stopped using the deprecated `remotectl` utility for creating certificates for the main `cockpit.service`, in favor of the much simpler `/usr/libexec/cockpit-certificate-ensure`. This version moves the remaining usage (containers and documentation) to that helper, and drops `remotectl`.
+
+If you use scripts, playbooks, or similar direct invocations of `remotectl`, please consider removing that: Cockpit creates or refreshes a self-signed certificate on startup automatically if required. If you must pre-create it, please move your scripts to call
+`/usr/libexec/cockpit-certificate-ensure`. However, please note that this script is *not* a stable and supported public interface.
+
+## Shell: Show package versions in 'About web console' modal
+
+![Screenshot from 2021-08-30 14-07-52about]({{ site.baseurl }}/images/252-shell-show-package-versions-in-about-web-console-modal.png)
+
+## Storage: Encryption is presented as a property of a Filesystem
+
+Cockpit used to make the encryption layer of a encrypted filesystem explicit: It had its own row in the content table and it would be referred to in other places in the UI.  Now the encryption layer is hidden away and Cockpit pretends encryption is a inherent property of a Filesystem.
+
+![screenshot of encryption is presented as a property of a filesystem]({{ site.baseurl }}/images/252-storage-encryption-is-presented-as-a-property-of-a-filesystem.png)
+
+
+## Try it out
+
+Cockpit 252 is available now:
+
+* [For your Linux system](https://cockpit-project.org/running.html)
+
+* [Cockpit Source Tarball](https://github.com/cockpit-project/cockpit/releases/tag/252)
+* [Cockpit Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit)
+* [Cockpit Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit)
+* [cockpit-machines Source Tarball](https://github.com/cockpit-project/cockpit-machines/releases/tag/251)
+* [cockpit-machines Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit-machines)
+* [cockpit-machines Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit-machines)
+* [cockpit-podman Source Tarball](https://github.com/cockpit-project/cockpit-podman/releases/tag/34)
+* [cockpit-podman Fedora 35](https://bodhi.fedoraproject.org/updates/?releases=F35&packages=cockpit-podman)
+* [cockpit-podman Fedora 34](https://bodhi.fedoraproject.org/updates/?releases=F34&packages=cockpit-podman)
```

Comment:
Please fill in the actual bodhi URLs from https://bodhi.fedoraproject.org/updates/?packages=cockpit and -machines and -podman